README.md

```diff
@@ -22,10 +22,10 @@ This will install `vulcanize` to `/usr/local/bin/vulcanize`.
 - `-p <arg>`|`--abspath <arg>`: use <arg> as the "webserver root", make all adjusted urls absolute
 - `--exclude <path>`: exclude a subpath from root. Use multiple times to exclude multiple paths. Tags to excluded paths are kept.
 - `--strip-excludes`: Exclude a subpath and remove any links referencing it.
-- `--implicit-strip`: Like specifying --strip-excludes for all the transitive dependencies of imports specified with `--exclude`.
 - `--inline-scripts`: Inline external scripts.
 - `--inline-css`: Inline external stylesheets.
 - `--strip-comments`: Strips all HTML comments not containing an @license from the document.
+- `--no-implicit-strip`: *DANGEROUS* Avoid stripping imports the transitive dependencies of imports specified with `--exclude`.
```

Comment:
Make this equivalent to the help output message. Add a note that this may allow for duplicate code inlining.